app/models/loan_response.rb

```diff
@@ -69,8 +75,12 @@ def has_boolean?
     field_attributes.include?(:boolean)
   end
 
+  def has_breakeven_table?
+    field_attributes.include?(:breakeven)
+  end
+
   def blank?
-    text.blank? && number.blank? && rating.blank? && boolean.blank? && url.blank?
+    text.blank? && number.blank? && rating.blank? && boolean.blank? && url.blank? && breakeven_data.blank?
```

Comment:
Are you sure `.blank?` is the right test here? We will need to make sure to normalize breakeven_data so it doesn't end up like [{x: [], y: []},[],[]] or something, which is not `blank?`.